src/api/artifact/artifact.service.ts

```diff
@@ -48,29 +57,42 @@ const updateArtifact = async (
   artifact_id: string,
   artifact_data: ArtifactUpdate
 ): Promise<artifact> => {
```

Comment:
Should promise an ArtifactResponse, since it's returning from addSignedUrlToArtifacts